manifests/p/pnpm/pnpm/8.13.1/pnpm.pnpm.installer.yaml

```
@@ -1,9 +1,9 @@
-# Created with WinGet Releaser v2 using Komac v1.11.0
+# Created with komac v2.0.4
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
 
 PackageIdentifier: pnpm.pnpm
 PackageVersion: 8.13.1
-InstallerType: portable
+InstallerType: exe

```

```suggestion
InstallerType: portable
```

I am fairly certain that pnpm executables are in fact not actual installers but simple, self-contained applications that should have their `InstallerType` set to `portable`.

---

It was caused by Komac